### PR TITLE
feat(nginx): add an option to set static ip for clusterIP service

### PR DESCRIPTION
**Pull request by Adibov**
I needed to create a clusterIP service for nginx which has static IP and there was no option to do so, therefore I created this MR to solve this issue.

### CODE REVIEW

**Comment by zyyw** [comment]
Thanks for contributing to harbor-helm! Please undo the change in this file:
* Chart.yaml

BTW, also I see there is a large diff on `README.md`. It seems that you format it a little bit with only one new entry for `expose.clusterIP.clusterIP`, right?

**Comment by zyyw** [comment]
Why your env is `static IP` specifically?

**Comment by Adibov** [comment]
> Thanks for contributing to harbor-helm! Please undo the change in this file:
> 
> * Chart.yaml
> 
> BTW, also I see there is a large diff on `README.md`. It seems that you format it a little bit with only one new entry for `expose.clusterIP.clusterIP`, right?

I roll-backed changes on `Chart.yaml` file. Also, you're right about the changes on `README.md` file. I only add documentation about `expose.clusterIP.clusterIP` field.

**Comment by Adibov** [comment]
> Why your env is `static IP` specifically?

Actually, we run self-hosted GitLab runners in a Kubernetes cluster in which we mounted the docker socket of all jobs' executors to their corresponding nodes. Therefore, we needed to access the harbor nginx service from nodes, and to achieve this functionality, we had to create a static IP service for harbor and a host alias in all nodes.

File: values.yaml [hunk 1]
```diff
@@ -54,6 +54,8 @@ expose:
   clusterIP:
     # The name of ClusterIP service
     name: harbor
+    # The ip address of the ClusterIP service (leave empty for acquiring dynamic ip)
+    clusterIP: ""
```
Comment (zyyw) [inline values.yaml:58]: could you please change `clusterIP.clusterIP` to `clusterIP.staticClusterIP`?
Thanks

Comment (Adibov) [reply values.yaml]: Done

File: templates/nginx/service.yaml [hunk 2]
```diff
@@ -13,6 +13,9 @@ metadata:
 {{- end }}
 spec:
   type: ClusterIP
+  {{- if .Values.expose.clusterIP.staticClusterIP }}
+  clusterIP: {{ .Values.expose.clusterIP.clusterIP }}
```
Comment (zyyw) [inline templates/nginx/service.yaml:17]: line 17. it should also be `.Values.expose.clusterIP.staticClusterIP` I suppose.

**Comment by zyyw** [comment]
@Adibov , please rebase main branch code and resolve the conflicts in `README.md` file.
Thank you!

File: templates/nginx/service.yaml (hunk 2 above)
Comment (Adibov) [reply templates/nginx/service.yaml]: You're totally right, Fixed.

**Comment by Adibov** [comment]
> @Adibov , please rebase main branch code and resolve the conflicts in `README.md` file. Thank you!

Rebased to the main branch and resolved conflicts. Any more comment would be welcomed :)

**Review by zyyw — APPROVED** [review]

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**